common/app/views/main.scala.html

```diff
@@ -27,34 +28,110 @@
     }
 }
 
-@immersiveMainMedia(page: GalleryPage) = {
-    <div class="new-gallery content tonal--tone-media immersive-main-media">
-        @defining(
-            page.item.content.trail.trailPicture.head
-        ) { picture =>
-            @fragments.image(
-                picture = picture,
-                classes = Seq("immersive-main-media__img"),
-                widths = MainMedia.immersive,
-                imageAltText = "",
-                isImmersiveMainMedia = true
-            )
+@immersiveMainMedia(page: ContentPage) = {
+    <div class="content content--immersive tonal tonal--@toneClass(page.item) immersive-main-media
+        @if(page.item.tags.isGallery) { new-gallery }
```

Comment:
I don't really know if this is a thing we do but as you're using `page.item.tags` multiple times, can you var it up as `t` or something as you only really care about `isArticle` or `isGallery` when reading the code?